src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -30,62 +29,45 @@
 )
 from ._docker_utils import get_login_credentials
 
+logger = get_logger(__name__)
 
-def acr_check_name(registry_name):
-    """Checks whether the container registry name is available for use.
-    :param str registry_name: The name of container registry
-    """
-    client = get_acr_service_client().registries
 
+def acr_check_name(client, registry_name):
     return client.check_name_availability(registry_name)
 
 
-def acr_list(resource_group_name=None):
-    """Lists all the container registries under the current subscription.
-    :param str resource_group_name: The name of resource group
-    """
-    client = get_acr_service_client().registries
-
+def acr_list(client, resource_group_name=None):
     if resource_group_name:
         return client.list_by_resource_group(resource_group_name)
-
     return client.list()
 
 
-def acr_create(registry_name,
+def acr_create(cmd,
+               client,
+               registry_name,
                resource_group_name,
                sku,
                location=None,
                storage_account_name=None,
                admin_enabled='false',
                deployment_name=None):
-    """Creates a container registry.
-    :param str registry_name: The name of container registry
-    :param str resource_group_name: The name of resource group
-    :param str sku: The SKU of the container registry
-    :param str location: The name of location
-    :param str storage_account_name: The name of storage account
-    :param str admin_enabled: Indicates whether the admin user is enabled
-    :param str deployment_name: The name of the deployment
```

Comment:
👍 Verified that this doesn't leave holes in the help?

Reply:
Yes and added the param entries for the missing ones as there were a few.

Reply:
Are comments no longer required or allowed and all help messages will come from param or help files?

Reply:
Comments in the custom.py commands are not desired. They will still work but we try and remove them since we're converting all the modules.  The help is more appropriately placed in the help.py file or as parameter entries in params.py.